quodlibet/quodlibet/qltk/cover.py

```diff
@@ -42,8 +42,16 @@ def __init__(self, title, fileobj, parent):
             width = int(width / 1.1)
             height = int(height / 1.1)
         else:
-            width = int(Gdk.Screen.width() / 1.8)
-            height = int(Gdk.Screen.height() / 1.8)
+            win = Gdk.Window.at_pointer()
+            disp = Gdk.Display.get_default()
+            if win[0] and disp:
+                mon = disp.get_monitor_at_window(win[0])
```

Comment:
this API is gtk 3.22+

Reply:
Ah, right. I'll make it compatible with older versions too.